apps/Oh My Posh/install-32

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+wget -O /tmp/oh-my-posh https://github.com/JanDeDobbeleer/oh-my-posh/releases/latest/download/posh-linux-arm
```

Comment:
I would still like this to be versioned (use the format like shown in the example install script from the documentation https://pi-apps.io/wiki/development/Creating-an-app/#making-an-install-script )

You should additionally add an auto-updater so that pi-apps can keep the users install of Oh My Posh up to date. I don't have great documentation written for how to do so but this commit should be a good start showing how to obtain the version number from github releases and how pi-apps expects to be able to update the version number of the application automatically https://github.com/Botspot/pi-apps/commit/fd584580618f627f6508f686a5a01b1b7cc67cfc